hoomd_validation/util.py

```diff
@@ -201,3 +202,154 @@ def make_simulation(
     sim.operations.add(quantity_writer)
 
     return sim
+
+
+def make_seed(job, sim_mode=None):
+    """Make a random number seed from a job.
+
+    Mix in the simulation mode to ensure that separate simulations in the same
+    state point run with different seeds.
+    """
+    # copy the job statepoint and mix in the simulation mode
+    statepoint = job.statepoint()
+    statepoint['sim_mode'] = sim_mode
+
+    return int(signac.job.calc_id(statepoint), 16) & 0xffff
+
+
+def plot_distribution(ax, data, xlabel, expected=None, bins=100):
+    """Plot distributions."""
+    import numpy
+
+    max_density_histogram = 0
+    sim_modes = data.keys()
+
+    range_min = min([min(x) for x in data.values()])
+    range_max = max([max(x) for x in data.values()])
```

Comment:
```suggestion
    range_min = min(min(x) for x in data.values())
    range_max = max(max(x) for x in data.values())
```
Use Python's generator syntax.